### DISCUSSION TITLE
Expand Chat Bot Support

### DISCUSSION THREAD

**Discussion by LucasAschenbach** [LucasAschenbach/prompster#5] [Ideas]
## Welcome 👋🤖

This discussion is about adding support for more chat bots to prompster. Below, you can find an up to date list of chat bots which prompster currently supports. I will update the list whenever a new bot has been added.  If you have a new chat bot you would like prompster to support, please comment here.

## Currently Supported

- ChatGPT
- Bard (Legacy)
- Gemini
- Claude
- Google AI Studio
- Groq
- OpenAI Playground
- Huggingface Chat
- Meta AI
- AgentGPT
- Poe
- nat.dev/chat
- you.com

_Last Updated: Apr 28th 2024_

**Comment by gmmedia** [comment]
https://nat.dev/chat would be another one.

**Reply by LucasAschenbach** [reply]
Thanks, completely forgot about that one! 👍 

Published a new version with it https://github.com/LucasAschenbach/prompster/releases/tag/v1.3.4

**Comment by malekinho8** [comment]
Hey @LucasAschenbach ,

Would love to have support for [scite](https://scite.ai/). It's a new AI tool that has come out for scientists/researchers to allow for finding relevant publications and research quickly, similar to Bing, but more fine-tuned for STEM.

**Comment by j3asd2** [comment]
Hi @LucasAschenbach 

first of all, thank you very much for the extension. I have tried many, and this one is one of my favorites—simple and useful!

It would be very useful to have support on https://huggingface.co/chat/ where it's free to create GPT assistants using many open-source models like Llama. 

Many thanks!

**Reply by LucasAschenbach** [reply]
Thanks for the suggestion, I published a new version https://github.com/LucasAschenbach/prompster/releases/tag/v1.4.4 with support for Huggingface Chat. Should be through review in less than 24h.

**Comment by paraskarmacharya** [comment]
With chatgpt adding a "/" option for 01 and websearch, the "/" is not working. Is there a way I can still make this work in ChatGPT.
I love your extension, it is the simplest to use.

Paras

**Comment by lonix1** [comment]
@LucasAschenbach Thanks for this extension!

It works for Google Gemini (https://gemini.google.com) but not for Google Gemini "AI Mode" (https://www.google.com/search?hl=en&udm=50).

That is probably the default for many users as it's accessible directly from the search pages. It's also a nice no-login alternative to Gemini for quick throw-away chats.

I assume it's very similar to the regular Gemini page. Please consider supporting it?
